pkgs/by-name/pa/parmetis/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  metis,
+  mpi,
+}:
+
+stdenv.mkDerivation {
+  pname = "parmetis";
+  version = "4.0.3";
+
+  src = fetchFromGitHub {
+    owner = "KarypisLab";
+    repo = "ParMETIS";
+    rev = "d90a2a6cf08d1d35422e060daa28718376213659";
+    hash = "sha256-22YQxwC0phdMLX660wokRgmAif/9tRbUmQWwNMZ//7M=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  enableParallelBuilding = true;
+  buildInputs = [ mpi ];
+
+  configurePhase = ''
+    tar xvf ${metis.src}
+    mv metis-* metis
+    make config metis_path=metis gklib_path=metis/GKlib prefix=$out
+  '';
+
+  meta = with lib; {
+    description = "MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices";
```

Comment:
```suggestion
    description = "MPI-based parallel library that implements a variety of algorithms";
```
thats a bit long

Reply:
I’d prefer to cut “that implements a variety of algorithms” so that some description of what the package actually does is retained…

Reply:
We also have [`longDescription`](https://nixos.org/manual/nixpkgs/stable/#var-meta-longDescription).

```nix
description = "Parallel Graph Partitioning and Fill-reducing Matrix Ordering";
longDescription = ''
  MPI-based parallel library that implements a variety of algorithms for
  partitioning unstructured graphs, meshes, and for computing fill-reducing
  orderings of sparse matrices.
  The algorithms implemented in ParMETIS are based on the multilevel
  recursive-bisection, multilevel k-way, and multi-constraint partitioning
  schemes
'';
```

Reply:
Thanks, I added it:)